pkgs/development/python-modules/octave-kernel/default.nix

```
@@ -0,0 +1,50 @@
+{
+  lib,
+  buildPythonPackage,
+  pythonOlder,
+  hatchling,
+  metakernel,
+  jupyter-client,
+  ipykernel,
+  nbconvert,
+  pytest,
+  fetchFromGitHub,
+}:
+
+buildPythonPackage rec {
+  pname = "octave_kernel";
+  version = "0.36.0";
+  pyproject = true;
+
+  disabled = pythonOlder "3.7";
+
+  src = fetchFromGitHub {
+    owner = "Calysto";
+    repo = "octave_kernel";
+    rev = "refs/tags/v${version}";
```

```suggestion
    tag = "v${version}";
```
New parameter for fetching tags